src/pages/index.js

```diff
@@ -118,13 +117,13 @@ function IndexPage() {
         <div className="container">
           <div className="row">
             <div className="col-sm-6 event_content">
-              <h3 className="next-cohort">Next Cohort</h3>
+              <h3 className="next-cohort">Surprising Update</h3>
               <div className="event_excerpt">
-                <p>Countdown until our next #VetsWhoCode Cohort gets started:</p>
+                <p>Launching something new Veterans Day. Be sure to apply!</p>
```

Comment:
Maybe think of wrapping in a conditional so it disappears after Veteran's Day? Also Veteran's Day with an apostrophe.

Reply:
I can work on that at a later date as that's not an non blocking change. Will fix the apostrophe.